### PR TITLE
Complete translate module essay + tests

### PR DESCRIPTION
**Pull request by vladhacksmile**
Complete translate module essay

Kharin Vladislav, P33121

### CODE REVIEW

File: src/course_bot/essay.clj [hunk 1]
```diff
@@ -38,8 +59,32 @@
...
     :assignment-error "ОШИБКА: почему-то не удается найти задание!"
-    :assignment-count "Количество заданий:"
-    :assignment-examples "Примеры: "}}})
+    :assignment-count "Количество заданий: "
+    :assignment-examples "Примеры: "
+    :first-essay-best "Первое эссе -- лучшее.\n\n"
+    :rank-1 "Ранг: %d, "
+    :essay-number-in-the-list-1 "Номер эссе в списке: #%d, "
+    :your-review-1 "Ваше ревью: %s "
+    :few-words-from-the-essay "\n(несколько слов из эссе: "
```
Comment (ryukzak) [inline src/course_bot/essay.clj:68]: not used?

**Review by ryukzak — APPROVED** [review]

**Comment by ryukzak** [comment]
Only essay?

File: src/course_bot/essay.clj (hunk 1 above)
Comment (vladhacksmile) [reply src/course_bot/essay.clj:68]: > Only essay?

Other modules in progress

Comment (vladhacksmile) [reply src/course_bot/essay.clj:68]: > not used?

Yeah, I'll clean it, because it was later combined into one text

**Pull request merged**